MetaMask Plugin/How to create a MetaMask Plugin

```diff
@@ -0,0 +1,26 @@
+Introduction
+
+1. Metamask plugin what’s new? 
+2. How this will improve or advantages over the extension.
+
+
+3.1 BreakDown of new Plugin APIs of Metamask. (Short Video/ Writeup)
+3.2 Explain Approach, and suggest concepts. 
+
+
+4. How to Switch from Extension to Plugin. 
```

Comment:
```suggestion
4. How to switch from extension to plugin. 
```

I'm not sure what you mean here, we're not switching from anything, it's about creating a plugin for Metamask.

Reply:
Apparently, the Community will build the plugins to make it more user-friendly. Hence switching from extension to plugin.

Reply:
Sorry, I still don't understand what you mean @thesachinmittal 

https://medium.com/metamask/introducing-the-next-evolution-of-the-web3-wallet-4abdf801a4ee

Metamask is building a plugin architecture that sits on top of a browser extension.